pkg/migrations/sql/preinstall/003-tables.sql

```diff
@@ -94,13 +94,16 @@ CREATE TABLE SCHEMA_CATALOG.metric (
     id SERIAL PRIMARY KEY,
     metric_name text NOT NULL,
     table_name name NOT NULL,
+    table_schema name NOT NULL DEFAULT 'SCHEMA_DATA',
+    series_table name DEFAULT NULL, -- NULL signifies the series table is named the same as metric table name
```

Comment:
Will update to be not null, should be easier to deal with the value.

I'm not sure about the `series_schema` just because I'm not sure why would we want to have series data outside a single schema. Is there a good reason?